### PR TITLE
`releases/compiler`: Support setting build tags in compiler

### PR DESCRIPTION
**Pull request by pete-woods**

### CODE REVIEW

File: releases/compiler/parallel_test.go [hunk 1]
```diff
@@ -28,23 +30,38 @@ func TestParallel_Compile(t *testing.T) {
 			Result:      &binary1,
 			Name:        "binary1",
 			Target:      "../..",
-			Source:      "./testing/compiler/internal/cmd",
```
Comment (pete-woods) [inline releases/compiler/parallel_test.go:31]: This was confusingly pointing at the wrong directory

File: releases/compiler/parallel_test.go [hunk 2]
```diff
@@ -28,23 +30,38 @@ func TestParallel_Compile(t *testing.T) {
...
+			Source:      "./releases/compiler/internal/cmd",
 			Environment: []string{"FOO=foo1", "BAR=bar1"},
 		})
 		c.Add(Work{
 			Result:      &binary2,
 			Name:        "binary2",
 			Target:      "../..",
-			Source:      "./testing/compiler/internal/cmd2",
```
Comment (pete-woods) [inline releases/compiler/parallel_test.go:38]: Same here

Comment (DominicLavery) [reply releases/compiler/parallel_test.go:38]: Copy and past issue i assume 😅

File: releases/compiler/parallel_test.go (hunk 1 above)
Comment (DominicLavery) [reply releases/compiler/parallel_test.go:31]: Copy and past issue i assume 😅

**Review by DominicLavery — APPROVED** [review]

**Pull request merged**